web/styles/globals.css

```diff
@@ -50,6 +50,15 @@
   font-display: swap;
 }
 
+@font-face {
+  font-family: "TWK Lausanne";
+  src: url("https://staging.world-id-assets.com/fonts/TWKLausanne-550.woff2")
```

Comment:
@andy-t-wang I assume there should be 400 font

Reply:
Also, is it possible to make it something like this?

```css
src: url('https://world-id-assets.com/fonts/TWKLausanne-400.woff2') format('woff2'),
url('https://staging.world-id-assets.com/fonts/TWKLausanne-400.woff2') format('woff2');

```

Reply:
hmm this doesn't work but yes added size 400 font